packages/oqamldebug/oqamldebug.0.9.5/opam

```diff
@@ -0,0 +1,12 @@
+opam-version: "1"
+maintainer: "sebastien.fricker@gmail.com"
+build: [
+  [ "qmake" "CONFIG-=app_bundle" "CONFIG+=release" "PREFIX_BIN=%{bin}%/" "CONFIG-=debug" "oqamldebug.pro" ]
+  ["%{make}%"]
+  ["%{make}%" "install" ]
+]
+remove: [
+  [ "qmake" "CONFIG-=app_bundle" "CONFIG+=release" "PREFIX_BIN=%{bin}%/" "CONFIG-=debug" "oqamldebug.pro" ]
+  [ "make" "uninstall" ]
```

Comment:
I don't believe that is the case. Using `make` rather than `"make"` will result in an indirection that is under opam's control. This mapping can be modified by the user with the `-m` or `--make` command line options to `opam install`.

I believe `"%{make}%"` will behave similarly but is

1. uglier
2. not what most other packages do

In short, `"foo"` is the literal string `"foo"` whereas `foo` is the opam variable value `foo`.

Using `make` instead of `"make"` is important for, e.g., FreeBSD if GNU make-isms are in use. On that platform, `make` resolves to `gmake` by default.